README.md

```diff
@@ -216,7 +219,7 @@ CREATE TABLE <TABLE NAME>
 By default you will have to fill in the table name. You can also specify the table name via the `-t` option:
 
 ```sql
-$ ./csv2db generate -f test/resources/201811-citibike-tripdata.csv -t STAGING
+$ ./csv2db generate -f test/resources/201811-citibike-tripdata.csv -t STAGING -o mysql
```

Comment:
I don't think this should change in the ReadMe. For the majority of use cases, running without the database type should be sufficient to generate the table DDL

Reply:
Thats fair. Sorry for changing this. Fixed in https://github.com/csv2db/csv2db/pull/51/commits/5a890c37c8575a6b6d95bc3f418fe8ef682e672f.